dev-tools/mage/gotool/linkcheck.go

```diff
@@ -27,3 +27,15 @@ func runGoLinkCheck(opts ...ArgOpt) error {
 }
 
 func (goLinkCheck) Path(path string) ArgOpt { return flagArgIf("-f", path) }
+
+func (goLinkCheck) MaxRetries(retries uint) ArgOpt {
```

Comment:
If you go with my suggestion in https://github.com/elastic/elastic-agent/pull/8063/files#r2071694627, maybe add a godoc here saying the argument value should be anything parseable by `time.ParseDuration`.